app/view/components/Loading.js

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Spinner from 'react-native-spinkit'
+import LottieView from 'lottie-react-native'
+import { StyleSheet, View, Text } from 'react-native'
+
+import i18n from 'i18n'
+import style from 'view/style'
+
+const lotties = [
```

Comment:
I'm working on an idea to have a Loading hook with api, state and error handling. And then cause of lottie power I prefer to use it instead of native and js animated spinners and loading. If you agree we can finalize it and pass these values as elegant suggested one. 
Also there is an idea about checking internet connection while using Loading so there will be a lot to go, I will put all these on trello if you're agree 👍